app/views/layouts/_navigation.html.erb

```diff
@@ -28,20 +28,20 @@
   
   <ul class="nav navbar-nav navbar-right">
     <% if user_signed_in? %>
-      <form class="navbar-form navbar-left visible-lg" role="search">
+      <form class="navbar-form navbar-left" role="search">
         <div class="form-group">
           <input type="text" class="form-control" id='agent-navigate' autocomplete="off" placeholder="Search">
           <%= image_tag "spinner-arrows.gif", :class => "spinner" %>
         </div>
       </form>
       
       <li class='job-indicator' role='pending'>
-        <%= link_to current_user.admin? ? jobs_path : '#', class: 'visible-lg' do %>
+        <%= link_to current_user.admin? ? jobs_path : '#', class: '.d-block' do %>
```

Comment:
I don't think the class name `.d-block` is defined anywhere, can we get rid of `class: '.d-block'`?

Reply:
yes, but I dont know how to remove the class without causing something went wrong error, no knowledge of codes 😅can you modify it right?